style.css

```diff
@@ -16,14 +16,15 @@ body {
 }
 
 .hero {
-	background-image: url('/assets/images/hero-banner.jpg');
+	background-image: url('../web-dev-playground/assets/images/hero-banner.jpg');
```

Comment:
This is telling it to go outside the root directory and then go back into the root directory. So it has the same effect as before, just with a bit more code.